--- Python-2.7.12/Doc/conf.py~	2016-06-25 16:49:30.000000000 -0500
+++ Python-2.7.12/Doc/conf.py	2016-07-20 13:40:03.438295455 -0500
@@ -135,6 +135,17 @@
 # Get LaTeX to handle Unicode correctly
 latex_elements = {'inputenc': r'\usepackage[utf8x]{inputenc}', 'utf8extra': ''}
 
+# Options for Texinfo output
+# --------------------------
+
+texinfo_documents = [
+    ('contents', 'python', 'Python Documentation', _stdauthor,
+     'Python', 'Python Documentation', 'Software Development'),
+]
+
+texinfo_show_urls = [
+    'footnote',
+]
 
 # Options for Epub output
 # -----------------------
