diff -urN Python-2.3-vanilla/Doc/Makefile Python-2.3/Doc/Makefile
--- Python-2.3-vanilla/Doc/Makefile	Mon Jul 28 23:11:34 2003
+++ Python-2.3/Doc/Makefile	Wed Jul 30 19:19:37 2003
@@ -150,8 +150,7 @@
 	html/ref/ref.html \
 	html/tut/tut.html \
 	html/inst/inst.html \
-	html/dist/dist.html \
-	html/whatsnew/$(WHATSNEW).html
+	html/dist/dist.html
 
 ALLHTMLFILES=$(INDEXFILES) html/index.html html/modindex.html html/acks.html
 
@@ -396,7 +395,6 @@
 	isilo/tut/tut.html \
 	isilo/inst/inst.html \
 	isilo/dist/dist.html \
-	isilo/whatsnew/$(WHATSNEW).html
 
 $(ISILOINDEXFILES): $(COMMONPERL) html/stdabout.dat perl/isilo.perl
 
@@ -408,8 +406,7 @@
 	isilo/python-ref.pdb \
 	isilo/python-tut.pdb \
 	isilo/python-dist.pdb \
-	isilo/python-inst.pdb \
-	isilo/python-whatsnew.pdb
+	isilo/python-inst.pdb
 
 isilo/python-api.pdb: isilo/api/api.html isilo/api/api.css
 	$(MKISILO) "-iPython/C API Reference Manual" \
@@ -506,7 +503,6 @@
 	$(WEBCHECKER) $(HTMLBASE)/tut/
 	$(WEBCHECKER) $(HTMLBASE)/dist/
 	$(WEBCHECKER) $(HTMLBASE)/inst/
-	$(WEBCHECKER) $(HTMLBASE)/whatsnew/
 
 fastwebcheck: $(ALLHTMLFILES)
 	$(WEBCHECKER) -x $(HTMLBASE)/api/
@@ -518,7 +514,6 @@
 	$(WEBCHECKER) -x $(HTMLBASE)/tut/
 	$(WEBCHECKER) -x $(HTMLBASE)/dist/
 	$(WEBCHECKER) -x $(HTMLBASE)/inst/
-	$(WEBCHECKER) -x $(HTMLBASE)/whatsnew/
 
 
 # Release packaging targets:
