diff -ru Python-2.6.2-orig/Doc/Makefile Python-2.6.2/Doc/Makefile
--- Python-2.6.2-orig/Doc/Makefile	2009-07-31 10:10:04.000000000 -0400
+++ Python-2.6.2/Doc/Makefile	2009-07-31 10:11:35.000000000 -0400
@@ -52,7 +52,7 @@
 	svn update tools/jinja2
 	svn update tools/pygments
 
-build: checkout
+build:
 	mkdir -p build/$(BUILDER) build/doctrees
 	$(PYTHON) tools/sphinx-build.py $(ALLSPHINXOPTS)
 	@echo
Only in Python-2.6.2/Doc: Makefile~
